config.py

```diff
@@ -210,13 +217,15 @@ class configuration:
   # A dictionary of the linker flags to use for each compiler type
   linker_flags = dict()
 
-  if (get_platform()[:6] == 'macosx'):
+  if ('macosx' in get_platform()):
     linker_flags['gcc'] = ['-fopenmp', '-dynamiclib', '-lpython2.7',
                            '-Wl,-install_name,' + get_openmoc_object_name()]
   else:
     linker_flags['gcc'] = ['-fopenmp', '-shared',
                            '-Wl,-soname,' + get_openmoc_object_name()]
 
+  linker_flags['clang'] = ['-fopenmp', '-shared',
+                         '-Wl,-soname,' + get_openmoc_object_name()]
```

Comment:
The `-soname` and `-shared` linker flags for clang should be changed to '-install_name' and `-dynamiclib` for clang on mac OS X just as with gcc.